docs/training/2025/webinar-series-2025Q2.md

```diff
@@ -48,51 +54,101 @@ Attendance is free of cost, but registration is required.
 
 ## Session details
 
-### Introduction to EESSI (5 May 2025)
+### Introduction to EESSI
 
-- Monday 5 May 2025 (13:30-15:30 CEST)
+- Monday 5 May 2025, 13:30-15:30 CEST (Zoom webinar)
 - speakers:
     - Richard Topuchain (University of Bergen)
     - Helena Vela (Do IT Now)
+- moderators:
     - Thomas Röblitz (University of Bergen)
-    - Kenneth Hoste (HPC - UGent)
-- outline
-    - Discover how EESSI (European Environment for Scientific Software Installation) is transforming the way scientific software is deployed and shared across HPC systems, cloud platforms, and even laptops. In this session, we’ll introduce the motivation behind EESSI, its architecture, and how you can start using a fully pre-built, modular software environment—no matter where you compute. Whether you're an HPC user, sysadmin, or developer, this webinar will show how EESSI can help you save time, improve reproducibility, and simplify your scientific workflows.
+    - Kenneth Hoste (Ghent University)
+
+#### Outline
+
+Discover how EESSI (European Environment for Scientific Software Installation) is transforming
+the way scientific software is deployed and shared across HPC systems, cloud platforms, and even laptops.
+
+In this session, we'll introduce the motivation behind EESSI, its architecture, and how you can start using
+a fully pre-built, modular software environment — no matter where you compute. Whether you're an HPC user,
+sysadmin, or developer, this webinar will show how EESSI can help you save time, improve reproducibility,
+and simplify your scientific workflows.
 
 ---
 
-### Introduction to CernVM-FS (12 May 2025)
+### Introduction to CernVM-FS
 
-- Monday 12 May 2025 (13:30-15:30 CEST)
+- Monday 12 May 2025, 13:30-15:30 CEST (Zoom webinar)
 - speakers:
-    - _more info soon_
-- outline
-    - _more info soon_
+    - Valentin Volkl (CERN)
+    - Kenneth Hoste (Ghent University)
+
+#### Outline
+
+CernVM-FS, the CernVM File System a.k.a. CVMFS (https://cernvm.cern.ch/fs), is a file distribution service
+that is particularly well suited to distribute software installations across a large number of systems world-wide
+in an efficient way.
+
+In this webinar, we will introduce you to CernVM-FS, show how to access existing CernVM-FS repositories (like [EESSI](https://eessi.io),
+cover some aspects specific to using CernVM-FS on HPC systems.
+
+It is intended for people with a background in HPC (system administrators, support team members, end users, etc.) who are new to CernVM-FS:
```

Comment:
Also here: should we limit to HPC? Or just state it targets system administrators _in general_? Again, sysadmins maintaining a cloud environment and willing to offer EESSI might also benefit - even if you may discuss _some_ HPC specific things?

Something like "While some HPC-specific considerations will be discussed, this session is relevant system administrator, support team member or end-user of any type of system"?

Reply:
I mean, I realize we will say _some_ HPC specific things, but I think the vast majority of the session will cover _general_ CVMFS?

Reply:
fixed in b906b870